src/hardhat.ts

```diff
@@ -0,0 +1,68 @@
+import { CHAINS } from './generated/chains';
+import { toUpperSnakeCase } from './utils/strings';
+import { Chain, HardhatEtherscanConfig, HardhatNetworksConfig } from './types';
+
+export function buildEnvVariables(): string[] {
+  return CHAINS
+    .filter((chain) => chain.explorer?.api?.key?.required)
+    .map((chain) => buildEtherscanApiKeyName(chain));
+}
+
+export function buildEtherscanApiKeyName(chain: Chain): string {
+  return `${toUpperSnakeCase(chain.alias)}_ETHERSCAN_API_KEY`;
```

Comment:
```suggestion
  return `ETHERSCAN_API_KEY_${toUpperSnakeCase(chain.alias)}`;
```

Reply:
So that it's like a type. My practical concern is that twenty of these will be in a `.env` file along with a few other variables, I'd like to be able to quickly skim through them to find `MNEMONIC` for example.